### ISSUE TITLE
SA1008: false positive when pattern matching a nested tuple

### ISSUE THREAD

**Issue by morsiu** [DotNetAnalyzers/StyleCopAnalyzers#3212]
The following code triggers SA1008 (Opening parenthesis should be spaced correctly):
``` csharp
public static bool Issue()
{
    return (false, (false, false)) switch
    {
        (_, (_, _)) => false,
        /*  ^--- SA1008 reports an issue here */
    };
}
```

In that case I think it should not as there should be a space after the preceding comma.
I got this on version 1.2.0-beta.205.

**Comment by sharwell** [comment]
Duplicate of #3198
